src/ocamlorg_frontend/pages/papers.eml

```diff
@@ -72,7 +73,7 @@ Layout.render
                 </div>
                 <% | _ -> %>
                 <table class="max-w-5xl lg:max-w-full align-top block lg:table">
-                    <thead class="bg-body-700 text-white text-left rounded-xl">
+                    <thead class="hidden lg:table-header-group bg-body-700 text-white text-left rounded-xl">
```

Comment:
```suggestion
                    <thead class="hidden lg:table-header-group bg-body-700 text-default text-left rounded-xl">
```

I have no idea why this table has `text-white` class. Hiding this on small screens is good, but let's fix this little problem as well.